tests/run-make/assert_imports_windows-msvc/expected.empty.txt

```diff
@@ -0,0 +1,122 @@
+
+File: empty.exe
+Format: COFF-x86-64
+Arch: x86_64
+AddressSize: 64bit
+Import {
+  Name: KERNEL32.dll
+  ImportLookupTableRVA: 0x207C0
+  ImportAddressTableRVA: 0x19000
```

Comment:
Is this an offset into the imported library or the executable itself? If the latter it would change between toolchain versions, right?

Reply:
They can change with rustc updates, but ignored explicitly `diff -u -I "File:" -I "ImportLookupTableRVA:" -I "ImportAddressTableRVA:" ...`